### PR TITLE
Story points documentation

### PR DESCRIPTION
**Pull request by CharlievWilliams**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,4 +53,19 @@ The reason you have to add this is so we can all add our own config files sepera
...
+### Story Points
+
+In agile we tend to estimate the length of time a task will take us to complete. This will allow us to better plan the amount of work we are capable of bringing in each sprint.
+
+To denote this we tend to use numbers from the Fibonacci sequence. 1, 2, 3, 5, 8, 13, 21. The logic here is that smaller tasks can accurately be estimated, however it is much harder to accurately estimate larger tasks. Hence, the larger difference between each number as it increases in size.
+
+During our first sprint, when we begin working on a ticket we should attempt to estimate the time it will take to complete. This will allow us to look back on this sprint and understand how many story points we are capable of completing. Then we can estimate future tasks before committing to completing it in a sprint.
+
+As a guideline, I think of the numbers as follows. I would encourage us to discuss and agree on this in the pull request for this Readme. Whatever we agree on, we should keep consistent across sprints, so we can properly visualise our productivity in graphs such as burndown charts.
+
```
Comment (CharlievWilliams) [inline README.md:68]: Feel free to share if you have a different opinion on this scale. We only have 3 sprints so it won't prove particularly essential. Alas, we must comply.

Comment (Tom-Goring) [reply README.md:68]: The scaling looks pretty spot on to me, I'm all for it.

**Review by Tom-Goring — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Maxwell2Gyamfi) [reply README.md:68]: Looks good

**Review by Maxwell2Gyamfi — APPROVED** [review]

**Review by Bm2pires — COMMENTED** [review]
looks great

**Pull request merged**